qa-covid19.planx-pla.net/manifest.json

```diff
@@ -63,16 +63,16 @@
   },
   "ssjdispatcher": {
     "job_images": {
-      "indexing": "quay.io/cdis/indexs3client:2021.04"
+      "indexing": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/indexs3client:2021.07"
```

Comment:
are all these job images in ECR?

Reply:
Yes all these job images are in ECR. After the release version is released, there is a job in release pipeline which pushes all the quay images to AWS ECR.